go/libkb/loaduser.go

```diff
@@ -46,6 +46,12 @@ func NewLoadUserByUIDArg(g *GlobalContext, uid keybase1.UID) LoadUserArg {
 	return arg
 }
 
+func (g *GlobalContext) LoadUser(uid keybase1.UID) (*User, error) {
```

Comment:
rename to LoadUserByUID?

but doesn't this add more to GlobalContext?

Also, would prefer if the GlobalContext methods were in libkb/globals.go otherwise hard to find.